File: Gradle/src/main/java/com/pqixing/modularization/git/GitCredential.java
package com.pqixing.modularization.git;

import com.pqixing.interfaces.ICredential;

import org.jetbrains.annotations.NotNull;

public class GitCredential implements ICredential {
    @NotNull
    @Override
    public String getUserName() {
        return null;
    }

    @NotNull
    @Override
    public String getPassWord() {
        return null;
    }
}

File: Q-gradle/src/main/java/com/pqixing/modularization/manager/GitCredential.kt
package com.pqixing.modularization.manager

import com.pqixing.Tools
import com.pqixing.modularization.utils.ICredential


class GitCredential : ICredential {

    override fun getUserName(): String {
        return ManagerPlugin.getExtends().docRepoUser
    }

    override fun getPassWord(): String {
        return ManagerPlugin.getExtends().docRepoPsw
    }
}
